assets/stylesheets/components/_modules.scss

```diff
@@ -0,0 +1,5 @@
+// contains each self-contained component in its own .scss partial
+
+@import 'alerts',
+        'tables',
+        'ig-confirmation';
```

Comment:
So this was something I was on the fence about. Brian wants the styles from embed brought into DDK, but I don't know if some of these styles would get used in any other scenario but embed - so I was thinking on prefixing inline giving styles with an `ig-` to demark them as embed styles. This line of code references the confirmation page of inline giving - a page that uses a very specific font size/weight found nowhere else.